### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by AvivCukierman** [khegazy/Bitcoin#2]
Write loss function:
E.g. three actions: Buy $100, sell $100, or hold
If buy: loss = $100 - (# bitcoins)*(bid price) - fees
If sell: loss = -$100 + (# bitcoins)*(ask price) - fees
If hold: 0
